backend/services/mgmt/v1alpha1/transformers-service/transformers.go

```diff
@@ -54,7 +56,7 @@ const (
 	Passthrough               Transformation = "passthrough"
 	Null                      Transformation = "null"
 	Invalid                   Transformation = "invalid"
-	Javascript                Transformation = "javascript"
+	Javascript                Transformation = "transform_javascript"
```

Comment:
Probably want to change this property to `TransformJavascript` as well as I think we will also want a javascript generator.

Reply:
This doesnt look like it was resolved.

Reply:
resolved

Reply:
I was referring to the property name itself.
```suggestion
	TransformJavascript                Transformation = "transform_javascript"
```